model-optimizer/automation/package_BOM.txt

```diff
@@ -981,3 +981,4 @@ requirements_kaldi.txt
 requirements_mxnet.txt
 requirements_onnx.txt
 requirements_tf.txt
+requirements_tf2.txt
```

Comment:
I think that we need to add `install_prerequisites/install_prerequisites_tf2.bat` and `install_prerequisites/install_prerequisites_tf2.sh` as well.

Reply:
Fixed.